app/controllers/users_controller.rb

```diff
@@ -1,13 +1,12 @@
 class UsersController < ApplicationController
-  has_filters %w{proposals debates budget_investments comments}, only: :show
+  has_filters %w{proposals debates budget_investments comments follows}, only: :show
 
   load_and_authorize_resource
   helper_method :author?
-  helper_method :author_or_admin?
```

Comment:
I can't really understand the reason to remove `author_or_admin?` method and its usage, can you explain a bit further? :)